partner_event/models/event_registration.py

```diff
@@ -40,9 +40,7 @@ def _update_attendee_partner_id(self, vals):
             Event = self.env["event.event"]
             # Look for a partner with that email
             email = vals.get("email").replace("%", "").replace("_", "\\_")
-            attendee_partner = Partner.search(
-                [("email", "=ilike", email)], limit=1, order="id"
-            )
+            attendee_partner = Partner.search([("email", "=ilike", email)], limit=1)
```

Comment:
Why removing the `order` part? That was done for avoiding extra queries for sorting the results.